x-pack/plugins/ingest_manager/public/applications/ingest_manager/index.tsx

```diff
@@ -229,15 +230,15 @@ const IngestManagerApp = ({
   const isDarkMode = useObservable<boolean>(coreStart.uiSettings.get$('theme:darkMode'));
   return (
     <coreStart.i18n.Context>
-      <CoreContext.Provider value={coreStart}>
+      <KibanaContextProvider services={{ ...coreStart }}>
```

Comment:
uses the newer KibanaContextProvider in place of CoreContext provider, which also exposes coreStart and allows use to use the <LinkToApp> component in the Siem/Endpoint app